content/guides/walkthroughs/multi-container-apps.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Run multi-container applications
+keywords: get started, quick start, intro, concepts
+description: Learn how to use Docker to run multi-container applications
+---
+
+If you've already completed the [How do I run a container?](./run-a-container.md) walkthrough, you learned that you must start each container individually. Imagine how great it would be if a tool could start multiple containers with a single command. That tool is Docker Compose.
+
+Before you start, [get Docker Desktop](../../get-docker.md).
+
+## Step 1: Get the sample application
+
+If you have git, you can clone the repository for the sample application. Otherwise, you can download the sample application. Choose one of the following options.
+
+{{< tabs >}}
+{{< tab name="Clone with git" >}}
+
+Use the following command in a terminal to clone the sample application repository.
+
+```console
+$ git clone https://github.com/docker/multi-container-app
+```
+
+{{< /tab >}}
+{{< tab name="Download" >}}
+
+Download the source and extract it.
+
+{{< button url="https://github.com/docker/multi-container-app/archive/refs/heads/main.zip" text="Download the source" >}}
+
+{{< /tab >}}
+{{< /tabs >}}
+
+The sample application is a simple todo application built using ExpressJS and Node.js. The application saves all todos in a MongoDB database. You don't need to know any of these technologies to continue with the walkthrough.
+
+![The sample app architecture](images/getting-started-multi-container.png?w=400)
+
+## Step 2: Dig into the Compose file
+
+View the files of the sample application. Notice that it has a `compose.yaml` file. This file tells Docker how to run your application. Open the `compose.yaml` file in a code or text editor to view what it contains.
+
+![Viewing the Compose file](images/getting-started-compose.png?w=400)
+
+## Step 3: Run the application
+
+To run the multi-container application, open a terminal and run the following commands. Replace `/path/to/multi-container-app/` with the path to your applications directory
+
+{{< include "open-terminal.md" >}}
+
+```console
+$ cd /path/to/multi-container-app/
+$ docker compose up -d
+```
+```console
+$ docker compose up -d
+```
+
+## Step 4: View the frontend and add todos
```

Comment:
could we add something about Compose Watch here?

Reply:
Yep, definitely! I was holding off until the sample app was updated, but it looks like they just merged it https://github.com/docker/multi-container-app/commit/7b11449d8a58a8d8dbb1a5954ed8a96b757a0b91